management/src/main/java/org/ehcache/management/providers/statistics/StandardEhcacheStatistics.java

```diff
@@ -61,24 +38,17 @@
 
 class StandardEhcacheStatistics extends ExposedCacheBinding {
 
-  private static final Map<String, SampleType> COMPOUND_SUFFIXES = new HashMap<String, SampleType>();
-
-  static {
-    COMPOUND_SUFFIXES.put("Count", SampleType.COUNTER);
-    COMPOUND_SUFFIXES.put("Rate", SampleType.RATE);
-    COMPOUND_SUFFIXES.put("LatencyMinimum", SampleType.LATENCY_MIN);
-    COMPOUND_SUFFIXES.put("LatencyMaximum", SampleType.LATENCY_MAX);
-    COMPOUND_SUFFIXES.put("LatencyAverage", SampleType.LATENCY_AVG);
-  }
-
   private final StatisticsRegistry statisticsRegistry;
+  private final StatisticsRegistryMetadata statisticsRegistryMetadata;
```

Comment:
`StatisticsRegistryMetadata` in tc-platform now encapsulate all the logic for querying stats and creating stats descriptors. This class is used both for standalone and clustered